recipes/libmatio/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+./configure --enable-mat73 --enable-extended-sparse --prefix="${PREFIX}" --with-zlib="${PREFIX}" --with-hdf5="${PREFIX}"
+make
+
+if [[ `uname` == 'Darwin' ]];
+then
+	eval DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib make check
+else
+	make check
```

Comment:
The tests are failing because of gcc version. This recipe was working for
me. The only difference is that I was using the gcc from conda.

On Wed, Mar 30, 2016, 5:54 PM jakirkham <notifications@github.com> wrote:

> In recipes/libmatio/build.sh
> <https://github.com/conda-forge/staged-recipes/pull/236#discussion_r57913648>
> :
>
> > @@ -0,0 +1,13 @@
> > +#!/bin/bash
> > +
> > +./configure --enable-mat73 --enable-extended-sparse --prefix="${PREFIX}" --with-zlib="${PREFIX}" --with-hdf5="${PREFIX}"
> > +make
> > +
> > +if [[ `uname` == 'Darwin' ]];
> > +then
> > +	eval DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib make check
> > +else
> > +	make check
>
> Does it? I mean a bunch of test are failing. I haven't looked at it too
> closely, but it is worth trying.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/236/files/46a0e2826bc4441b34c89bd27e667eed9540df11#r57913648>
>